nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const app = express();
+
+app.use(express.urlencoded({
+    extended: true
+}))
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/:feature", (req, res) => {
+    const feature = req.params.feature;
+    const parameters = Object.keys(req.query);
+    const numbers = Object.values(req.query).flat().map(number => parseFloat(number));
+
+    function falsyCheck() {
```

Comment:
NIce!

Reply:
You can also make falsyCheck as a global function and then call it inside get and/or post to not repeat code twice

Reply:
Thanks for your nice comments. 

I have to get query to create falsyCheck function. How can I make it global?

Reply:
and then here instead of declaring function just call it 
const falsyCheck = getFalsyCheck(parameters, numbers);